### ISSUE TITLE
Document directory-related advanced settings

### ISSUE THREAD

**Issue by agitter** [martinpacesa/BindCraft#76]
I was having problems getting BindCraft to find my AlphaFold 2 weights files when they were not in the default location. I eventually found
https://github.com/martinpacesa/BindCraft/blob/23c1bbf541763aac80d35ecefd9c86fbeedd821d/bindcraft.py#L38-L40

and realized I could use `af_params_dir` in the `--advanced` settings to control that. The path is relative to the `bindcraft.py` directory, not the current working directory, which I did not anticipate.

The issue is to recommend documenting `af_params_dir`, `dssp_path`, and `dalphaball_path` in the readme.

Full error message in case anyone else encounters it:
```
WARNING: 'model_1_multimer_v3' not found
WARNING: 'model_2_multimer_v3' not found
WARNING: 'model_3_multimer_v3' not found
WARNING: 'model_4_multimer_v3' not found
WARNING: 'model_5_multimer_v3' not found
Stage 1: Test Logits
Traceback (most recent call last):
  File "/var/lib/condor/execute/slot1/dir_937028/./BindCraft-1.1.0/bindcraft.py", line 109, in <module>
    trajectory = binder_hallucination(design_name, target_settings["starting_pdb"], target_settings["chains"],
  File "/var/lib/condor/execute/slot1/dir_937028/BindCraft-1.1.0/functions/colabdesign_utils.py", line 97, in binder_hallucination
    af_model.design_logits(iters=50, e_soft=0.9, models=design_models, num_models=1, sample_models=advanced_settings["sample_models"], save_best=True)
  File "/usr/local/apps/pyenv/versions/miniforge3-23.11.0-0/envs/bindcraft-conda/lib/python3.9/site-packages/colabdesign/af/design.py", line 354, in design_logits
    self.design(iters, **kwargs)
  File "/usr/local/apps/pyenv/versions/miniforge3-23.11.0-0/envs/bindcraft-conda/lib/python3.9/site-packages/colabdesign/af/design.py", line 348, in design
    self.step(lr_scale=lr_scale, num_recycles=num_recycles,
  File "/usr/local/apps/pyenv/versions/miniforge3-23.11.0-0/envs/bindcraft-conda/lib/python3.9/site-packages/colabdesign/af/design.py", line 214, in step
    self.run(num_recycles=num_recycles, num_models=num_models, sample_models=sample_models,
  File "/usr/local/apps/pyenv/versions/miniforge3-23.11.0-0/envs/bindcraft-conda/lib/python3.9/site-packages/colabdesign/af/design.py", line 89, in run
    model_nums = self._get_model_nums(num_models, sample_models, models)
  File "/usr/local/apps/pyenv/versions/miniforge3-23.11.0-0/envs/bindcraft-conda/lib/python3.9/site-packages/colabdesign/af/design.py", line 71, in _get_model_nums
    ns = [ns[n if isinstance(n,int) else ns_name.index(n)] for n in models]
  File "/usr/local/apps/pyenv/versions/miniforge3-23.11.0-0/envs/bindcraft-conda/lib/python3.9/site-packages/colabdesign/af/design.py", line 71, in <listcomp>
    ns = [ns[n if isinstance(n,int) else ns_name.index(n)] for n in models]
IndexError: list index out of range
```

**Comment by martinpacesa** [comment]
Yes, initially it was hard coded in the settings but as the pipeline was aimed mostly at people with little computational expertise who rarely change any settings, I wanted it to be present in the default paths. People who know what they are doing can quickly find the right variable anyways ;)

**Comment by rukibuki** [comment]
this was also a problem for me, I eventually fixed it (in version 1.0.0) by changing:

advanced_settings["af_params_dir"] = os.path.realpath('')   

to: 

advanced_settings["af_params_dir"] = '/AF_main_path'

and then it worked. 

I support that it ought to be mentioned in the readme, I  think multiple ppl will run into this problem.
